ibis/backends/sqlite/__init__.py

```diff
@@ -338,6 +338,17 @@ def _generate_create_table(self, table: sge.Table, schema: sch.Schema):
 
         return sge.Create(kind="TABLE", this=target)
 
+    def _in_memory_table_exists(self, name: str) -> bool:
+        ident = sg.to_identifier(name, quoted=self.compiler.quoted)
+        query = sg.select(STAR).from_(ident).limit(0).sql(self.dialect)
```

Comment:
I wish this could be a generic implementation, but the exception handling is different for every backend, and it didn't seem worth the trouble to make *that* an overridable method.

Reply:
Looked into most of the remaining backends.